385/Assets/Scripts/RopeSystem.cs

```diff
@@ -114,6 +114,15 @@ public bool IsRopeConnected()
 		return (RopeAnchorPoint != null);
     }
 
+    // gets anchor point rope is attached to

```

Comment:
very optional, but try using the `///` comments in C#. they allow for Intellisense to provide hints to what the methods do, kinda like javadoc.